build.zig

```diff
@@ -105,20 +105,6 @@ const Maker = struct {
         e.want_lto = m.enable_lto;
         return e;
     }
-
-    fn lib(m: *const Maker, name: []const u8, src: []const u8, deps: []const *Compile) *Compile {
```

Comment:
I'm not familiar with llama.cpp's zig codebase, what is the motivation for these changes?

Reply:
Oh I guess they're reverted in the latter commits. Maybe rebase `--autosquash`

Reply:
Sorry for the extremely confusing commit history - I went through quite a few experiments before figuring out that we can essentially just replace stdenv gcc compiler with MinGw when cross compiling for Windows. Initially I tried using the Zig build system to achieve this but using the CMake build system is much smarter.

Reply:
You are more than welcome to just squash all the commits 👍

Reply:
Fair